packages/itwinui-react/src/core/Surface/Surface.tsx

```diff
@@ -29,7 +29,47 @@ const getSurfaceElevationValue = (elevation: SurfaceProps['elevation']) => {
   }
 };
 
+// ----------------------------------------------------------------------------
+// Surface.Header component
+
+export type SurfaceHeaderProps = React.ComponentPropsWithRef<'div'>;
+
+const SurfaceHeader = React.forwardRef<HTMLDivElement, SurfaceHeaderProps>(
+  (props, ref) => {
+    const { children, className, ...rest } = props;
+    return (
+      <div className={cx('iui-surface-header', className)} ref={ref} {...rest}>
+        {children}
+      </div>
+    );
+  },
+);
+
+// ----------------------------------------------------------------------------
+// Surface.Body component
+
+export type SurfaceBodyProps = React.ComponentPropsWithRef<'div'>;
+
+const SurfaceBody = React.forwardRef<HTMLDivElement, SurfaceBodyProps>(
+  (props, ref) => {
+    const { children, className, ...rest } = props;
+    return (
+      <div className={cx('iui-surface-body', className)} ref={ref} {...rest}>
+        {children}
+      </div>
+    );
+  },
+);
+
 export type SurfaceProps = {
+  /**
+   * TODO: Not sure how to describe this prop
+   */
+  layout?: boolean;
```

Comment:
maybe `useLayout` and `isPadded` are better?
For layout prop you could write "Aligns children vertically when set to true" or smth. Simply describe what it does.